mesheryctl/internal/cli/root/model/init.go

```diff
@@ -0,0 +1,401 @@
+package model
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"slices"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/meshery/schemas"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/mod/semver"
+)
+
+var initModelCmd = &cobra.Command{
+	Use:   "init [model-name]",
```

Comment:
```suggestion
	Use:   "init",
```
to keep consistency with other commands